assignment1/kthInLinkedList_test.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for kthInLinkedList()."""
+
+from kthInLinkedList import *
+import unittest
+
+class kthInLinkedListTest(unittest.TestCase):
+    def setUp(self):
+        self.n = 15
+        self.head = Node(self.n - 1)
```

Comment:
It may be worth either having a comment to explain you are creating a list with values [14 13 12 10 .. 0] or do this using a private methd on the test class (and make the name be enough to understand, e.g., `self.head = createReverseSequenceOfLength(15)`).